MonoZelda/Collision/Collidables/ItemCollidable.cs

```diff
@@ -22,6 +31,18 @@ public ItemCollidable(Rectangle bounds, ItemList itemType)
         this.itemType = itemType;
     }
 
+    public void PlaySound()
+    {
+        if (itemSoundEffects.ContainsKey(itemType))
+        {
+            itemSoundEffects[itemType].Invoke();
+        }
+        else
+        {
+            SoundManager.PlaySound("LOZ_Get_Item", false);
+        }
+    }
```

Comment:
From what I can tell the behavior of this method is "play the heart sound if this item is a heart, the rupee sound if this item is a rupee, and the get item sound otherwise." This seems like a reletively complex way to code that behavior but there could be benefits that I'm not seeing.